sklearn/discriminant_analysis.py

```diff
@@ -607,18 +610,27 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Discriminant Analysis
     """
 
-    def __init__(self, priors=None, reg_param=0., store_covariances=False,
-                 tol=1.0e-4):
+    def __init__(self, priors=None, reg_param=0., store_covariance=False,
+                 tol=1.0e-4, store_covariances=None):
         self.priors = np.asarray(priors) if priors is not None else None
         self.reg_param = reg_param
         self.store_covariances = store_covariances
+        self.store_covariance = store_covariance
         self.tol = tol
 
+    @property
+    @deprecated("Attribute covariances_ was deprecated in version"
+                " 0.19 and will be removed in 0.21. Use "
+                "covariance_ instead")
+    def covariances_(self):
+        return self.covariance_
+
     def fit(self, X, y):
         """Fit the model according to the given training data and parameters.
 
             .. versionchanged:: 0.19
-               *store_covariance* has been moved to main constructor.
+               *store_covariances* has been moved to main constructor as
```

Comment:
Improvement: `store_covariances` is not a parameter of `fit` anymore. Use the `store_covariance` parameter in the constructor instead.

Also I would use double-backticks (monospace in reST) for the parameter names rather than asterisk (italic in reST).